app/helpers/raster/georaster.py

```diff
@@ -1,49 +1,59 @@
 # -*- coding: utf-8 -*-
-
-import shputils
 from os.path import dirname
+import logging
 from struct import unpack
 
-import logging
-log = logging.getLogger('alti')
+from .shputils import SHPUtils
+
+log = logging.getLogger(__name__)
 
-# cache of GeoRaster instances in function of the layer name
-_rasters = {}
-_rasterfiles = {}
 RESOLUTION = 2
 
 
-def get_raster(sr):
-    global _rasters
-    result = _rasters.get(sr, None)
-    if result is None:
-        result = GeoRaster(_rasterfiles[sr])
-        _rasters[sr] = result
-        log.debug("GeoRaster for {} has been added in the cache".format(repr(sr)))
-    return result
-
-
-def init_rasterfiles(datapath, supported_spatial_references):
-    global _rasterfiles
-    _rasterfiles = {
-        # LV03
-        21781: datapath + 'swissalti3d/kombo_2m_regio/index.shp',
-        # LV95
-        2056:  datapath + 'swissalti3d/kombo_2m_regio_lv95/index.shp'
-        # for other projections, results are reprojected from LV95 model
-    }
-    try:
-        # this is currently the same as doing it for all rasterfiles, but if we support one day another projection and
-        # don't want to preload it, we have the possibility to do so.
-        for sr in supported_spatial_references:
-            get_raster(sr)
-            log.info('Preloading raster for spatial reference: %s' % sr)
-
-    except Exception as e:
-        log.error('Could not initialize raster files. Make sure they exist in the following directory: %s (Exception: %s)' % (datapath, e))
+class GeoRasterUtils(object):
+    # pylint: disable=logging-format-interpolation
+    # pylint: disable=broad-except
```

Comment:
Move the pylint comment to the broad exception. In this case because the exception is re-raise catching broad exception is acceptable and don't need to be documented but having the pragma next to it is better.

Reply:
done